index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'react-native-toast-message' {
+  import React from 'react';
+  const Toast: {
+    show(options: {
+      type?: 'success' | 'error' | 'info';
```

Comment:
Let's have `type` required and as `string`, since there can be custom types as well
```suggestion
      type: string;
```

Reply:
@calintamas Wouldn't it be the case to extend the proptypes again to add the types? Since success, error and information are the standards that would facilitate in most cases.

Reply:
sounds good to me. updating my comment above